### DISCUSSION TITLE
Crashes at Nibelheim

### DISCUSSION THREAD

**Discussion by Blutsabbath** [julianxhokaxhiu/FFNx#606] [Q&A] [answered]
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/12700285/FFNx.log)

Game crashes since I arrived at Nibelheim (after Canyon, not the flashback). Every time I zone from town map into a building the game crashes. Starting the game without mods was also no solution. Steam file check was also ok. Anyone an idea what the problem could be?

**Comment by julianxhokaxhiu** [comment, marked as answer]
Please update your FFNx version to the latest available. If that doesn't help consider using our Discord support channels. Thanks :)
